app/code/Magento/Theme/Model/ResourceModel/Theme/Collection.php

```diff
@@ -17,6 +17,11 @@ class Collection extends \Magento\Framework\Model\ResourceModel\Db\Collection\Ab
      */
     const DEFAULT_PAGE_SIZE = 6;
 
+    /**
+     * @var string
+     */
+    protected $_idFieldName = 'theme_id';
```

Comment:
Please change protected to private due to Magento [Technical Guidelines](https://devdocs.magento.com/guides/v2.3/coding-standards/technical-guidelines.html)